numba/tests/test_builtins.py

```diff
@@ -1387,7 +1387,7 @@ def foo(x):
             self.assertEqual(got, expected)
 
     @TestCase.run_test_in_subprocess
-    def test_experimental_warning(self):
+    def test_no_experimental_warning(self):
```

Comment:
Now CI has run and this change has been effectively confirmed... what do you think to just deleting this whole test? I don't think there's a need to check that there is no warning raised here any more than in any other piece of code where no warning is expected?

Reply:
Done!

Reply:
Oops, my bad!